src/views/Results.vue

```diff
@@ -14,8 +14,9 @@
       <Facet v-for="facet in facets" v-bind:facet="facet" v-bind:key="facet" />
     </div>
     <div v-bind:class="contentClass" v-if="this.status.loading == false">
+      <SearchMetadata v-model:searchterm="searchterm" v-model:hits="hits" />
```

Comment:
I don't object to this, but want to make sure we're choosing it intentionally. This would put the SearchMetadata display alongside any chosen facets, rather than above them (which is where the other status-report-like UI elements would appear).

Reply:
I agree with what you are saying in that it should be intentional to determine where things display, but I'm choosing for now to wait until we have real facets displaying to make this decision. If Adam lands his code first, we can definitely address this. For now I'm pretending that non-displaying component doesn't exist as I'm confident it has changed way to much for me to make assumptions about here.

Reply:
Resolving this...